### ISSUE TITLE
Add Mean Intersection over Union (mIoU) metric

### ISSUE THREAD

**Issue by jshin1394** [google/metrax#61]

**Comment by jshin1394** [comment]
Using the Keras [implementation](https://www.tensorflow.org/api_docs/python/tf/keras/metrics/MeanIoU) as basis, add the metric to metrax.

**Comment by aymuos15** [comment]
Since this is a good first issue, could I possibly take this up to get familiar with the library? Or is this already implemented?

Thanks :)

**Comment by aymuos15** [comment]
@jshin1394 

Sorry to tag, but is this still in the pipeline?
